src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -1842,6 +1844,41 @@ class VariantSampleMatrix[T](val hc: HailContext, val metadata: VariantMetadata,
 
   def storageLevel: String = rdd.getStorageLevel.toReadableString()
 
+  def setVAattribute(path: String, key: String, value: String): VariantSampleMatrix[T] = {
```

Comment:
4 overloaded methods is too much noise -- bring these down to 2 methods max. Also, invalid camel case here.

VariantSampleMatrix has gotten SUPER cluttered.